README.md

```diff
@@ -23,29 +23,27 @@ Everything we create is open source and freely distributed. A full listing of re
 - Building tools and pipelines for reproducible data analysis using Python
 - Implementing reproducible pipelines using both High Performance Computing (HPC) and Elastic Cloud Computing (ECC)
 - Writing clear documentation and scholarly publications
-- Teaching and consulting on data science and best practices for reproducibility (e.g. [Software and Data Carpentries](https://carpentries.org/), [BrainHack Global](https://brainhack.org/global2023/), [FAES](https://w.faes.org/home-new))
+- Advocating for open science best practices for reproducibility (e.g. [Software and Data Carpentries](https://carpentries.org/), [BrainHack Global](https://brainhack.org/global2023/), [FAES](https://w.faes.org/home-new))
 - Creating, expanding, and supporting [FAIR](https://en.wikipedia.org/wiki/FAIR_data) data standards in biomedical science (*e.g.* [BIDS](http://bids.neuroimaging.io/), [NWB](https://www.nwb.org/) [OME](https://www.openmicroscopy.org/), [GO](https://geneontology.org/))
 
-This position has minimal supervisory responsibilities, however motivated and productive individuals will have the opportunity to quickly assume a more senior role involving supervision and oversight of additional projects in parallel.
-
 ### Necessary Qualifications
 
-- A recent graduate degree (PhD preferred) in a STEM field or equivalent knowledge and experience
+- A recent undergraduate degree in a STEM field or equivalent knowledge and experience
 - Strong coding skills in Python
 - Experience with version control and continuous integration with git and github
```

Comment:
Continuous integration is not a common experience and could scare away people who would otherwise be quite good in a role like this.

```suggestion
- Experience with version control using Git and GitHub
```